### PR TITLE
quick fix

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: azure-core/azure/core/pipeline_client.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def __init__(self, base_url, config=None, pipeline=None):
             raise ValueError("Config is a required parameter")
         self.config = config
         self.base_url = base_url
-        self.pipeline = pipeline or self._build_pipeline(config, credentials)
```
Comment (annatisch) [inline azure-core/azure/core/pipeline_client.py]: But there is no `self._build_pipeline()` method?
Maybe we should just make `pipeline` a required parameter?

Comment (xiangyan99) [reply azure-core/azure/core/pipeline_client.py]: This is a good idea. I will make it required.

**Pull request merged**